Tools/AP_Periph/networking.h

```diff
@@ -54,7 +54,20 @@ class Networking_Periph {
 #endif // HAL_PERIPH_NETWORK_NUM_PASSTHRU
 
     AP_Networking networking_lib;
+
+#if AP_NETWORKING_BACKEND_PPP
+    AP_Int8 ppp_port;
+    AP_Int32 ppp_baud;
+#endif
 };
 
+#ifndef AP_PERIPH_NET_PPP_PORT_DEFAULT
```

Comment:
why aren't these at the top?

Reply:
you mean top of the class in header? any particular reason?

Reply:
because all other ifndef's are at the top everywhere else!

Reply:
ahh, I see what you mean, fixed